index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>My portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <nav class="flex-toolbar">
+      <div class="logo-name"><p>Jesus Molina</p></div>
+      <div class="menu-flex">
+        <img id="menu" src="resources/Menu-button/Enabled.svg" alt="A menu button" />
+      </div>
+    </nav>
+
+    <section class="headline-section">
+      <div id="primary-text">
+        <p>Hey There.<br> I'm Jesus</p>
```

Comment:
- [ ] This is the main title and it would be better wrapped in a proper `<h1>` tag. Kindly change that and take a look at [this article](https://www.reliablesoft.net/h1-tag/) for a better understanding.

Reply:
Fixed